src/api/recordlinker/assets/demo_data.json

```diff
@@ -90,7 +191,39 @@
                 "type": "eCR"
             },
             "link_score": 0.82,
-            "linked": false
+            "linked": false,
```

Comment:
Won't we need all of these to be `"linked": null` as the initial state so that it shows in the table?

Reply:
+1

Reply:
Good catch! I fixed and added two other patients to help with testing but they won't appear on the queue page.